ts/src/config.ts

```diff
@@ -102,6 +102,10 @@ export interface Config extends AuthenticationConfig {
   // too large.
   // https://nodejs.org/dist/latest-v9.x/docs/api/timers.html#timers_settimeout_callback_delay_args.
   serverBackoffCapMillis?: number;
+
+  // Use test-cloudprofiler.sandbox.googleapis.com instead of
+  // cloudprofiler.googleapis.com when this is set to true.
+  useTestApi?: boolean;
```

Comment:
Maybe expose it as baseApiUrl instead so that if there are other environment we want to test in future we can use the same flag?

Reply:
Done!